src/loader-components/ExampleTemplate.module.scss

```diff
@@ -49,10 +50,27 @@
     display: flex;
     outline: none;
     padding: 0.625rem;
+
+    &.is-selected {
+      background-color: var(--terra-dev-site-example-template-selected-code-toggle-background-color, #666565);
+      color: var(--terra-dev-site-example-template-selected-code-toggle-color, #ebebeb);
+    }
+
+    &:hover {
+      background-color: var(--terra-dev-site-example-template-hover-code-toggle-background-color, #b8b8b8);
+
+      &.is-selected {
+        background-color: var(--terra-dev-site-example-template-hover-selected-code-toggle-background-color, #424141);
+      }
+    }
+  }
+
+  .item[data-focus-styles-enabled='true']:focus {
+    outline: 2px dashed var(--terra-dev-site-example-template-keyboard-focus-outline, #000);
+    outline-offset: -2px;
   }
 
   .chevron {
-    color: var(--terra-dev-site-example-template-chevron-color, #000);
```

Comment:
Should this stay?

Reply:
I removed this originally so that the chevron could would always match the text color in the case of the tab being highlighted or selected, re-adding it and making a different text variable for it would be very simple if that's more appropriate? It would add quite a few more variables though.